rust-book-old/src/toolchain/katana/interact.md

```diff
@@ -0,0 +1,321 @@
+### install starkli
+
+```console
+curl https://get.starkli.sh | sh
+## open a new terminal
+starkliup
```

Comment:
Currently we should use `starkliup -v 0.1.20` to be compatible with katana `0.4.4`. Newer version are not supported due to RPC incompatibility.